frontend/src/components/Admin/AdminShoutouts/AdminShoutouts.tsx

```diff
@@ -83,8 +86,19 @@ const AdminShoutouts: React.FC = () => {
     }
   }, [allShoutouts, earlyDate, lastDate, view]);
 
+  const fetchImages = useCallback((shoutouts: Shoutout[]) => {
+    shoutouts.forEach((shoutout) => {
+      if (shoutout.images?.length) {
+        ImagesAPI.getEventProofImage(shoutout.images[0]).then((url) => {
```

Comment:
Since this isn't an event proof image, we should probably have a separate `getShoutoutImage` function. Same with the upload image function. However, the logic behind all these functions will be the same and will be repetitive, so I'm thinking maybe we could just have an overall `getImage` and `uploadImage` function in the Images API. @andrew032011 What do you think? This will also affect the coffee chat images, so I could make this change in PR #642.

Reply:
Thanks for bringing this up! Yeah, I agree we should unify it into one API. getImage and estimate (or uploadImage). @patriciaahuang do you want to repurpose your Coffee Chat image API PR to instead make the generalizable API? I thiiink it should be as simple as some renaming.